aten/src/ATen/native/cuda/CuFFTPlanCache.h

```diff
@@ -389,12 +390,17 @@ class CuFFTParamsLRUCache {
 
     // Miss
     // remove if needed
+    // bug related to cuFFT plan cache max size has been fixed
+    // in CUDA 10. Hence, when compiling with CUDA 10, just
+    // don't do the erase.
+    #if CUDA_VERSION < 10000
```

Comment:
I don't think that we should make the cache silently unconditionally grow. If I remember correctly, some cuFFT caches may allocate GPU memory. It would make sense to actually have a reasonable default size. Furthermore, now you are saying with CUDA 10 the python methods that configs the cache size return successfully, but doesn't actually change it!

I think a better fix is to: (1) use a much much larger default number for CUDA 10, and (2) update the documentation.

Reply:
Thanks! Sorry for being late to the party. I didn't notice this one...